lib/msf/core/payload/python/meterpreter_loader.rb

```diff
@@ -32,11 +32,7 @@ def initialize(info = {})
           'MeterpreterTryToFork',
           'Fork a new process if the functionality is available',
           default: true
-        ),
-        OptBool.new(
-          'PythonMeterpreterDebug',
```

Comment:
Maybe add an alias to keep supporting this option?

Reply:
Example:

https://github.com/rapid7/metasploit-framework/blob/d88c5752d2b58a6f250671fdc4abd5c261cc195c/modules/exploits/windows/smb/psexec.rb#L75-L78